tck/running_the_tck.asciidoc

```diff
@@ -44,6 +44,11 @@ To enable the tests in your project you need to add the following dependency to
     <version>6.9.9</version>
     <scope>test</scope>
 </dependency>
+<dependency>
+    <groupId>org.awaitility</groupId>
+    <artifactId>awaitility</artifactId>
+    <version>3.1.6</version>
+</dependency>
```

Comment:
Not sure this is needed. Awaitility should be pulled in as a transitive dependency of the TCK, so the user shouldn't have to declare it themselves (unless I'm missing something here?)